core/sql/ustat/hs_globals.cpp

```diff
@@ -8472,6 +8472,23 @@ Lng32 HSGlobalsClass::groupListFromTable(HSColGroupStruct*& groupList,
 
     // Initialize the pointer to the group list we will build.
     groupList = NULL;
+    
+    // if showstats for a native hbase table, need to check if the schema _HBASESTATS_ exist
+    if (strcmp(hstogram_table->data(), "TRAFODION.\"_HBASESTATS_\".SB_HISTOGRAMS") == 0)
```

Comment:
Are there any other examples of this kind of bug besides native HBase access? What happens if we do SHOWSTATS on a native Hive table, for example, but the "_HIVESTATS_" schema has not yet been created? (This kind of question is why I was hoping for a solution where we interpreted the error code from cursor103.open() instead.)